### PR TITLE
add task 1 checksum

### PR DESCRIPTION
**Pull request by 20jmx02**
Task 1 Checksum ready to merge into the main branch.
Code is approved.

### CODE REVIEW

**Comment by 20jmx02** [comment]
Sorry I forgot to put it into the "Aufgabe_1" folder.

**Comment by Destinyphoenix** [comment]
> Sorry I forgot to put it into the "Aufgabe_1" folder.

then move it there with a commit pls

File: Blatt_3/Checksum.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package Blatt_3;
```
Comment (Destinyphoenix) [inline Blatt_3/Checksum.java:1]: we currently don't use packages

Comment (Destinyphoenix) [reply Blatt_3/Checksum.java:1]: or use  package Blatt_3.Aufgabe_1;

File: Blatt_3/Checksum.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            sum += lastChar;
+            // remove last character
+            x = x / base;
+        }
+        return sum;
+    }
+    public static String checksumHex(String hex) {
+        // convert hex-string to integer
+        int intHex = Integer.parseInt(hex, 16);
+        return Integer.toHexString(checksum(intHex, 16));
```
[the remark this answers is not in the document]
Comment (20jmx02) [reply Blatt_3/Checksum.java:35]: Richtig muss 10 sein habe ich im nächsten Commit geändert.

File: Blatt_3/Checksum.java (hunk 1 above)
Comment (20jmx02) [reply Blatt_3/Checksum.java:1]: I use Blatt_3.Aufgabe_1 in the next commit.

**Pull request closed without merging**